src/test/java/seedu/address/testutil/TypicalMeetings.java

```diff
@@ -83,6 +83,6 @@ public static MeetingBook getTypicalMeetingBook() {
     }
 
     public static List<Meeting> getTypicalMeeting() {
-        return new ArrayList<>(Arrays.asList(MEETING_A, MEETING_B));
+        return new ArrayList<>(Arrays.asList(MEETING_B, MEETING_A));
```

Comment:
Was this switched because of the dates?

Reply:
Yes, it was. Meeting B's From value is 'earlier' than A's. Since we are comparing the result of deleting a meeting from a model that is auto sorted with `getTypicalMeeting().get(INDEX_FIRST_PERSON.getZeroBased())`, the ArrayList in the code snippet should be sorted too.